admin-dev/themes/new-theme/js/pages/product/services/images.ts

```diff
@@ -23,25 +23,27 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+/* eslint-disable max-len */
+
 import Router from '@components/router';
 
 const router = new Router();
 const {$} = window;
 
-export const getProductImages = async (productId) => {
+export const getProductImages = async (productId: number): Promise<JQuery.jqXHR<any>> => {
```

Comment:
```suggestion
export const getProductImages = async (productId: number): Promise<FetchResponse> => {
```

I think we should generalize this typing everywhere in our services (except if we want it to be even more accurate like `Promise<CustomizedFetchResponse>`)

Reply:
We cant..